cloud-formation/membership-app.cf.json

```diff
@@ -102,35 +102,28 @@
                     "Fn::Join":["\n", [
                         "#!/bin/bash -ev",
 
-                        "CONF_DIR=/membership/frontend-1.0-SNAPSHOT/conf",
+                        "CONF_DIR=/etc/frontend",
 
-                        "/opt/features/ssh-keys/initialise-keys-and-cron-job.sh -l -b github-team-keys -t Membership-and-Subscriptions || true",
-
-                        "adduser --system --home /membership --disabled-password membership",
-
-                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/frontend-1.0-SNAPSHOT.tgz /tmp"]]},
-
-                        "tar xvz -C /membership -f /tmp/frontend-1.0-SNAPSHOT.tgz",
+                        {"Fn::Join":["", ["aws --region ", { "Ref": "AWS::Region" }, " s3 cp s3://membership-dist/", { "Ref" : "Stack" }, "/", { "Ref" : "Stage" }, "/frontend/frontend_1.0-SNAPSHOT_all.deb /tmp"]]},
```

Comment:
Wow, that makes everything look much better! Hopefully the changes I made in 1318ec5 are all valid.